lib/npm/test.js

```diff
@@ -28,6 +28,10 @@ function authorName(author) {
 }
 
 function test(context, next) {
+  if (context.options.script) {
+    // This handles the citgm.js case
+    context.module.script = context.options.script;
```

Comment:
can you please send a different PR to fix this. Potentially fixing in CITGM so the behavior is the same and this catch is not neccessary